server.py

```diff
@@ -48,6 +49,18 @@
 serving_data = False
 clients = 0
 
+# Keep track of different runs
+pod_runs = 0
+server_start_time = datetime.now()
+run_start_time = datetime.now()
+logged_sensors = ["velocity","acceleration","rpm_scaled","temperature_inside","temperature_outside","temperature_electronics","blank"]
+logging_sensor_data = False
+for s in logged_sensors:
+    f = open('static/data/' + str(s) + ".csv",'w')
```

Comment:
Just because of the code I used for the graph. It was designed to receive the data this way and didn't think it was worth changing that.